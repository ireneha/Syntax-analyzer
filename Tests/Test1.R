# Test1.R

# This program tests basic statements and expressions. 

source ("List.R")

main <- function () {
  x <- as.integer (readline ());
  y <- as.integer (readline ());
  q <- 0;
  r <- x;
  while (r >= y) {
    q <- q + 1;
    r <- r - y;
  }
  print (q);
  print (r);
}
