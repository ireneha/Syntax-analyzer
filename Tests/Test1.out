Script started on 2021-11-28 19:21:16-0600
brb0164@faculty% R

R version 3.4.4 (2018-03-15) -- "Someone to Lean On"
Copyright (C) 2018 The R Foundation for Statistical Computing
Platform: i686-pc-linux-gnu (32-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> source ("Test1.R")
> main ()
32
5
[1] 6
[1] 2
> main ()
4430
6
[1] 738
[1] 2
> main ()
2147483647
1073741823
[1] 2
[1] 1
> q ()
Save workspace image? [y/n/c]: n
brb0164@faculty% exit

Script done on 2021-11-28 19:27:38-0600
